markets/perps-market/contracts/modules/MarketConfigurationModule.sol

```diff
@@ -5,37 +5,27 @@ import {IMarketConfigurationModule} from "../interfaces/IMarketConfigurationModu
 import {SettlementStrategy} from "../storage/SettlementStrategy.sol";
 import {PerpsMarketConfiguration} from "../storage/PerpsMarketConfiguration.sol";
 import {PerpsMarket} from "../storage/PerpsMarket.sol";
+import {PerpsPrice} from "../storage/PerpsPrice.sol";
 import {OrderFee} from "../storage/OrderFee.sol";
 
 contract MarketConfigurationModule is IMarketConfigurationModule {
     using PerpsMarket for PerpsMarket.Data;
+    using PerpsPrice for PerpsPrice.Data;
 
-    function addSettlementStrategy(
+    uint256 private constant _MIN_SETTLEMENT_DELAY = 1;
+
+    function setSettlementStrategy(
         uint128 marketId,
         SettlementStrategy.Data memory strategy
-    ) external override returns (uint256 strategyId) {
+    ) external override {
         PerpsMarket.load(marketId).onlyMarketOwner();
 
-        strategy.settlementDelay = strategy.settlementDelay == 0 ? 1 : strategy.settlementDelay;
-
-        PerpsMarketConfiguration.Data storage config = PerpsMarketConfiguration.load(marketId);
-        strategyId = config.settlementStrategies.length;
-
-        config.settlementStrategies.push(strategy);
-        emit SettlementStrategyAdded(marketId, strategy);
-    }
+        strategy.settlementDelay = strategy.settlementDelay == 0
```

Comment:
Maybe a tiny bit better to do `strategy.settlementDelay < _MIN_SETTLEMENT_DELAY` if we update to `_MIN_SETTLEMENT_DELAY` to something other than 1